core/modules/backend/controllers/DashboardController.php

```diff
@@ -30,6 +32,14 @@ class DashboardController extends ControllerBase
      */
     public function indexAction()
     {
+        $analytic = new Analytic();
+        // We check if user authorization
+        if ($analytic->checkAccessToken()) {
+            $this->view->isLogged = true;
+        } else {
+            $this->view->isLogged = false;
+        }
```

Comment:
`$this->view->isLogged = $analytic->checkAccessToken()` without needs of any `if`